hana/packages.sls

```diff
@@ -40,3 +40,15 @@ python3-shaptools:
         attempts: 3
         interval: 15
     - resolve_capabilities: true
+
+{# If venv-salt-minion is installed, then we make shaptools available on this environment #}
+{% if salt['pkg.version']('venv-salt-minion') %}
+shaptools_available_in_salt_bundle:
+  file.symlink:
+    - name: /usr/lib/venv-salt-minion/lib/python3.10/site-packages/shaptools
```

Comment:
Maybe slightly improve the solution with getting the proper `site-package` path with more flexible way:
```
python3 -c "import sysconfig as s; print(s.get_paths().get('purelib'))"
```
and for the bundle:
```
/usr/lib/venv-salt-minion/bin/python -c "import sysconfig as s; print(s.get_paths().get('purelib'))"
```

Reply:
Thanks for the suggestions Victor. I've adapted the SLS now.

Reply:
IMO the hardcoded version is easier to read. `/usr/bin/python3` will always be 3.6 for SLE 15 and I don't think we'll change the location of the venv before we fix the extension.

Reply:
But we can bump the version of the python inside the bundle. Maybe hardcoded python version of the distro is ok, but in case of upgrading the python in the bundle we would have to adopt it again.

Reply:
Before we change the Python version in the Bundle, we ought to fix the problem with installing shaptools inside of it properly. This PR is not the proper solution, it's an ugly hack that we should drop again as soon as we can.

Reply:
I see Alex's point, as mentioned, the final fix for this formula should be done once we enhance the extending mechanism for the Salt Bundle.

OTOH I think I would prefer to keep this hack as it is right now (by not hardcoding the python version) just in case we bump the Python version inside the Salt Bundle in the future (even when having already an enhanced bundle extending mechanism) but this formula has not been adapted/released yet, which will potentially cause another L3.